docs/can-guides/commitment/recipes/todomvc-with-steal/14-component-edit/todo-list.js

```diff
@@ -1,13 +1,13 @@
 // components/todo-list/todo-list.js
 import { StacheElement, type } from "can";
 import view from "./todo-list.stache";
-import Todo from "~/models/todo";
+import Todo, { TodoList } from "~/models/todo";
```

Comment:
I just realized we’re not exporting `Todo` on its own… should we add that so this can just be `{ Todo, TodoList }`?

Reply:
this isn't using can-demo-models... The local `todo` module exports default `Todo`.